<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RhythmEditor</class>
 <widget class="QDialog" name="RhythmEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rhythm Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="l_OriginalTaps">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ori&amp;ginal taps:</string>
         </property>
         <property name="buddy">
          <cstring>original</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="original"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="l_ButtonBlock">
       <item>
        <layout class="QHBoxLayout" name="l_Tempo">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Te&amp;mpo:</string>
           </property>
           <property name="buddy">
            <cstring>tempo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="tempo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="value">
            <number>120</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="autoTempo">
         <property name="text">
          <string>Determine &amp;automatically</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="dotted">
         <property name="text">
          <string>Detect &amp;dotted</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="tapButton">
         <property name="text">
          <string>&amp;Tap</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quantizeButton">
         <property name="text">
          <string>&amp;Quantize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetButton">
         <property name="text">
          <string>&amp;Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="l_QuantizedTaps">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Quantized taps:</string>
         </property>
         <property name="buddy">
          <cstring>quantized</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="quantized"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>original</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>81</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>quantized</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RhythmEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RhythmEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
